game/camera.h

```diff
@@ -117,6 +127,7 @@ class Camera final {
     bool                  lbEnable      = false;
     bool                  inertiaTarget = true;
     Mode                  camMod        = Normal;
+    MarvinMode            camMarvinMod  = M_Normal;
```

Comment:
Are `Mode` and  `MarvinMode` complementary or mutably exclusive?
If exclusive - it make sense to have them into one enum.

Reply:
This is a bit unclear to me. If a dialog is started in F5/F7 mode dialog cam takes over and afterwards F5/F7 takes over again. So there has to be some memory which cam was active before.
 
I moved the marvinMode enum to `mainwindow.cpp` and use `solveCameraMode` routine  to return the respective camera. So only one enum in `camera.cpp` with marvin cams included.

Reply:
> f a dialog is started in F5/F7 mode dialog cam takes over and afterwards F5/F7 takes over 

I didn't know that - resolved